tests/test_MultilabelBinaryPolicy.py

```diff
@@ -0,0 +1,794 @@
+    ground_truth = [
+        False,
+        False,
+        True,
+        True,
+        True,
+        True,
+        True,
+        True,
+        # Mutli-label Test cases
+        False,
+        False,
+        True,
+    ]
+    result = policy.negative_examples("1")
+    assert_array_equal(ground_truth, result)
+
+
+def test_less_inclusive_policy_negative_examples_2(digraph, features_1d, labels):
+    policy = LessInclusivePolicy(digraph, features_1d, labels)
+    ground_truth = [
+        True,
+        True,
+        False,
+        False,
+        False,
+        False,
+        False,
+        False,
+        # Mutli-label Test cases
+        True,
+        False,
+        False,
+    ]
+    result = policy.negative_examples("2")
+    assert_array_equal(ground_truth, result)
+
+
+def test_less_inclusive_policy_negative_examples_3(digraph, features_1d, labels):
+    policy = LessInclusivePolicy(digraph, features_1d, labels)
+    ground_truth = [
+        True,
+        True,
+        True,
+        True,
+        False,
+        True,
+        True,
+        True,
+        # Mutli-label Test cases
+        True,
+        True,
+        False,
+    ]
+    result = policy.negative_examples("2.1.1")
+    assert_array_equal(ground_truth, result)
+
+
+def test_siblings_policy_negative_examples_1(digraph, features_1d, labels):
+    policy = SiblingsPolicy(digraph, features_1d, labels)
+    ground_truth = [
+        False,
+        False,
+        True,
+        True,
+        True,
+        True,
+        True,
+        True,
+        # Mutli-label Test cases
+        False,
+        True,
+        True,
+    ]
+    result = policy.negative_examples("1")
+    assert_array_equal(ground_truth, result)
+
+
+def test_siblings_policy_negative_examples_2(digraph, features_1d, labels):
+    policy = SiblingsPolicy(digraph, features_1d, labels)
+    ground_truth = [
+        True,
+        True,
+        False,
+        False,
+        False,
+        False,
+        False,
+        False,
+        # Mutli-label Test cases
+        True,
+        True,
+        False,
+    ]
+    result = policy.negative_examples("2")
+    assert_array_equal(ground_truth, result)
+
+
+def test_siblings_policy_negative_examples_3(digraph, features_1d, labels):
+    policy = SiblingsPolicy(digraph, features_1d, labels)
+    ground_truth = [
+        False,
+        False,
+        False,
+        True,
+        False,
+        False,
+        True,
+        True,
+        # Mutli-label Test cases
+        False,
+        False,
+        True,
+    ]
+    result = policy.negative_examples("2.1")
+    assert_array_equal(ground_truth, result)
+
+
+def test_siblings_get_binary_examples_1d_1(digraph, features_1d, labels):
+    policy = SiblingsPolicy(digraph, features_1d, labels)
+    ground_truth_x = [
+        1,
+        2,
+        9,
+        10,
+        3,
+        4,
+        5,
+        6,
+        7,
+        8,
+        10,
```

Comment:
```suggestion
```
I think we should not have an example as both positive and negative at the same time. I would like to suggest updating the code in the `get_binary_examples` method to avoid labeling as negative something that is already in the positive examples.